Deepgram/Clients/Analyze/v1/Client.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Copyright 2021-2024 Deepgram .NET SDK contributors. All Rights Reserved.
+// Use of this source code is governed by a MIT license that can be found in the LICENSE file.
+// SPDX-License-Identifier: MIT
+
+using Deepgram.Models.Analyze.v1;
+using Deepgram.Models.Authenticate.v1;
+
+namespace Deepgram.Clients.Analyze.v1;
+
+/// <summary>
+/// Implements version 1 of the Analyze Client.
+/// </summary>
+/// <param name="apiKey">Required DeepgramApiKey</param>
+/// <param name="deepgramClientOptions"><see cref="DeepgramClientOptions"/> for HttpClient Configuration</param>
+public class Client(string apiKey, DeepgramClientOptions? deepgramClientOptions = null)
+    : AbstractRestClient(apiKey, deepgramClientOptions)
+{
```

Comment:
Will address the TODOs for Logging here since I haven't thought about how to address it yet:
https://github.com/deepgram/deepgram-dotnet-sdk/issues/210
https://github.com/deepgram/deepgram-dotnet-sdk/issues/219